packages/cardano-services/test/Asset/CardanoTokenRegistry.test.ts

```diff
@@ -35,10 +35,42 @@ const mockTokenRegistry = async (handler: () => { body: unknown; code?: number }
   };
 };
 
+const testDescription = 'test description';
+const testName = 'test name';
+
 describe('CardanoTokenRegistry', () => {
   const notValidAssetId = Cardano.AssetId('0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef');
   const validAssetId = Cardano.AssetId('f43a62fdc3965df486de8a0d32fe800963589c41b38946602a0dc53541474958');
 
+  describe('parseTokenMetadataServiceRecord', () => {
+    it('complete attributes', () =>
+      expect(
+        parseTokenMetadataServiceRecord({
+          decimals: { value: 23 },
```

Comment:
Missing `subject`. Best to add it to the second test case too.

Reply:
434b443db should solve this